wasm/jit/mmap_other.go

```diff
@@ -0,0 +1,8 @@
+//go:build !darwin && !linux
```

Comment:
ps I think it is fine to rename this to _windows and wait for someone to ask for something besides darwin and linux. we would likely want CI if that were the case anyway